src/commands/buildCommands/trn.js

```diff
@@ -0,0 +1,70 @@
+const child_process = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const paths = require('./config/paths');
+const mkdirp = require('mkdirp');
+
+const { allLocalPoTrnsWithFallbacks$, localTrns$ } = require('../txCommands/util/index');
+const localeCodes = require('./config/locales');
+
+const MODULES_PATH = path.resolve(paths.repoRoot, 'src/trns/modules/');
+
+const writeTrnModules = messagesByLocale => ({ filename, msgids }) => {
+	localeCodes.forEach(localeCode => {
+		if (!messagesByLocale[localeCode]) {
+			messagesByLocale[localeCode] = {};
+		}
+		// create object of trns for current localeCode
+		const trnsForLocale = msgids.reduce((trns, msgid) => {
+			trns[msgid] = messagesByLocale[localeCode][msgid];
+			return trns;
+		}, {});
+		// write the object to a file
+		const relPath = path.relative('./src/', filename);
+		const destFilename = path.resolve(
+			MODULES_PATH,
+			localeCode,
+			`${relPath}.json`
+		);
+		const destDirname = path.dirname(destFilename);
+		mkdirp.sync(destDirname);
+		fs.writeFileSync(
+			destFilename,
+			`${JSON.stringify(trnsForLocale, null, 2)}\n`
+		);
+	});
+};
+
+const componentTrnDefinitions$ = localTrns$.map(trnsFromFile => ({
+	filename: path.resolve(paths.repoRoot, `./${trnsFromFile[0].file.replace(/\.jsx?$/, '')}`),
```

Comment:
the second arg here doesn't need `./`, so it can just be

```js
filename: path.resolve(paths.repoRoot, trnsFromFile[0].file.replace(/\.jsx?$/, '')),
```